pkgs/development/libraries/tbb/2021_8.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, gcc11Stdenv
+, fetchFromGitHub
+, fetchpatch
+, cmake
+}:
+
+let
+  stdenv = gcc11Stdenv;
```

Comment:
Same as above, why not use default compiler here? Using non-default compiler will probably break many more things.

Reply:
Same reason as above. It is not supported. And the build failed.